projects/core/src/lib/components/toc/toc/toc.component.css

```diff
@@ -8,8 +8,16 @@
   padding-right: var(--toc-side-padding);
 }
 
+.toc-tree ::ng-deep .mat-tree-node {
+  height: fit-content !important;
```

Comment:
Do not use !important unless really needed, in this case I don't think its needed at all